infra/variables.tf

```diff
@@ -55,7 +55,7 @@ variable "init" {
 
 variable "image_version" {
   type        = string
-  default     = "v1.4.1"
+  default     = "v1.5.0"
```

Comment:
thought: Normally I might suggest we update Avocano in a separate PR, but I recognize this is carrying a change required for this PR to work.